.github/ISSUE_TEMPLATE/user-story-template.md

```diff
@@ -25,6 +25,27 @@ Then
 - [ ] Checklist item
 - [ ] Meets the DOD
 
+**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)
+- [ ] User story is included
+- [ ] User role and type are identified
+- [ ] Acceptance criteria are included
+- [ ] Wireframes are included (if required)
+- [ ] Does this User Story need stakeholder approval?
```

Comment:
What does ticking the box mean here?

Reply:
Again, what does ticking the box mean here? Does it mean that the story needs stakeholder approval, or does it mean that we have approval from all the required stakeholders?

Reply:
I get it now. This one I will remove probably. We are not really asking Stakeholder approval a User stories level and doesn't seems feasible too.